### DISCUSSION TITLE
Offline or internet required?

### DISCUSSION THREAD

**Discussion by maia** [JeffSteinbok/hass-dreo#94] [Q&A] [answered]
Does the Dreo integration connect to the fans locally or via the internet? I'm looking into purchasing a fan with smart home support but would want to control it locally, without contacting a server somewhere outside of my home to do so. Unfortunately I cannot find any information about the connectivity of the device (Wifi? Zigbee? Thread?) and the control. Thanks!

**Comment by mikewren** [comment]
In the pre-purchase phase and wondering the same thing.  It appears per the onboarding instructions that an internet connection and authentication against their cloud is required at least for initial configuration. Not sure if day-to-day usage is direct from Home Assistant direct to device via Wi-Fi or via cloud?

**Comment by JeffSteinbok** [comment, marked as answer]
This requires internet and controls fans via the Dreo servers.
